administrators.rst

```diff
@@ -267,12 +263,17 @@ Transfer Source. This is the only type of location that will be allowed inside a
 Dataverse space.
 
 Within this location, the relative path can be used to store a query string,
-which is used to interrogate the Dataverse Search API (for example, q=* will
-return all datasets).
+which is used to interrogate the Dataverse Search API. The q parameter is a 
+general search parameter. The ’subtree’ parameter can be used to indicate a 
+sub-dataverse. For example:
+   q=*
+   subtree=“Archivematica”
```

Comment:
Formatting this as a [literal block](http://docutils.sourceforge.net/docs/user/rst/quickref.html#literal-blocks) will make it render nicely. Change to:

```
The ’subtree’ parameter can be used to indicate a 
sub-dataverse. For example::

   q=*
   subtree=“Archivematica”

will return all datasets contained within...
```
Note the double colon marking the start of the literal block and the extra whitespace around the block.

Reply:
yep, thanks!